Code/Source/svFSI/all_fun.cpp

```diff
@@ -574,10 +583,10 @@ double integ(const ComMod& com_mod, const CmMod& cm_mod, const faceType& lFa, co
   if (nNo != com_mod.tnNo) {
     if (com_mod.ibFlag) {
       if (nNo != com_mod.ib.tnNo) {
-        throw std::runtime_error("Incompatible vector size in Integ");
+        throw std::runtime_error("Incompatible vector size in IntegS 1");
```

Comment:
I do not think that these runtime errors (IntegS 1 vs. IntegS 2) would be clear for a user. Can we slightly modify the error messages?

Reply:
@MatteoSalvador We need to distinguish between user errors and developer errors. A user error should be caught when reading in the solver XML file, reading mesh, initial, boundary condition data, etc.

 I'm guessing that this is a developer error. It should still provide more information of course, like, I don't know, maybe the name of the face for which this is failing.

Reply:
Adding more descriptive error message. Not sure if this is descriptive enough. Anybody (a developer) who gets this message will likely still need to look at this bit of code.